internal/cli/atlas/backup/snapshots/create.go

```diff
@@ -55,7 +55,7 @@ func (opts *CreateOpts) Run() error {
 	if err != nil {
 		return commonerrors.Check(err)
 	}
-	return opts.Print(r.GetActualInstance())
```

Comment:
Those were risky as we were returning subset of JSON as oposed to whole JSON in manual SDK